database/aoai-proxy.sql

```diff
@@ -323,6 +353,21 @@ CREATE TABLE aoai.event_catalog_map (
 
 ALTER TABLE aoai.event_catalog_map OWNER TO admin;
 
+--
+-- Name: metric; Type: TABLE; Schema: aoai; Owner: admin
+--
+
+CREATE TABLE aoai.metric (
+    event_id character varying(50) NOT NULL,
+    metric_timestamp date NOT NULL,
+    api_key uuid NOT NULL,
+    request_id integer NOT NULL,
```

Comment:
We're using an incrementing ID rather than `uuid`? Do we run the risk of hitting number caps?

Reply:
Should be unlikely to hit the cap, if it does we've got other issues to tackle!